README.md

```diff
@@ -1,11 +1,12 @@
-# CakePHP Framework
-
+<p align="center"><a href="https://cakephp.org" target="_blank"><img src="https://cakephp.org/img/trademarks/logo-1.jpg"></a></p>
```

Comment:
But then I can't align the image in the center, which then would look ugly if it's left aligned 😕